src/hooks/posters.ts

```diff
@@ -71,24 +71,53 @@ export const usePosters = (posterHallId: string) => {
   const fuseVenues = useMemo(
     () =>
       new Fuse(filteredPosterVenues, {
-        keys: ["poster.title", "poster.authorName", "poster.categories"],
+        keys: [
+          "name",
+          "poster.title",
+          "poster.authorName",
+          "poster.categories",
+        ],
+        threshold: 0.2, // 0.1 seems to be exact, default 0.6: brings too distant if anyhow related hits
+        ignoreLocation: true, // default False: True - to search ignoring location of the words.
+        findAllMatches: true,
       }),
     [filteredPosterVenues]
   );
 
   const searchedPosterVenues = useMemo(() => {
-    if (!searchQuery)
-      return filteredPosterVenues.slice(0, displayedPostersCount);
+    const normalizedSearchQuery = searchQuery.trim();
+
+    if (!normalizedSearchQuery) return filteredPosterVenues;
 
     return fuseVenues
-      .search(searchQuery)
-      .slice(0, displayedPostersCount)
+      .search({
+        //@ts-ignore
```

Comment:
What seems to be the issue here? Can we remove this and fix it easily?

Reply:
IIRC it was about untyped 
```
 .map((x) =&gt; ({            $or: [...
```
below

Reply:
This seems to have been fixed. Not sure what commit it was in. Probably one of @mike-lvov 's

Resolving.